config/locales/en.yml

```diff
@@ -820,67 +814,57 @@ funding:
       strand2:
         question: Do you currently have a product or service that is operational and that is being used by the type of people and communities you plan to serve in future?
-        unsure: >
-          <p>This programme is specifically designed to help organisations that:</p>
-          <ul>
-            <li>Were founded after the year 2000, i.e were ‘born in the digital age’</li>
-            <li>Have a digitally-enabled product or service that has at least a few current users - this strand is not for proposals that are at the idea or prototype stage.</li>
-          </ul>
-    ineligible:
-      title: Sorry, you’re ineligible
-      strand1: >
-        <p>Sorry, you’re probably not eligible for Strand 1 funding.</p>
-        <p>This strand of the Digital Fund is specifically designed to help organisations that:</p>
-        <ul>
-          <li>Are medium or large in size</li>
-          <li>Have structures and ways of working which were developed before the internet became widespread</li>
-          <li>Are looking to make a big change to the way they work</li>
-          <li>Primarily help people in the UK</li>
-        </ul>
-        <p>There are, however, other possibilities.</p>
-        <p>First, the Big Lottery Fund’s <a href="/funding/programmes">other funding programmes</a> are still open to you, and fully embrace digitally enabled ideas, so long as they help people and communities to thrive.</p>
-        <p>Next, this is just the first phase of a two year programme. <a href="https://en-gb.facebook.com/BigLotteryFund/">Follow us</a> or keep checking back to find out what’s next.</p>
-        <p>Finally, if you’re from a small charity or small community group, we’re planning on offering you some <a href="/funding/programmes/digital-fund/assistance">free digital support from 2019</a></p>
-      strand2: >
-        <p>Sorry, you’re probably not eligible for Strand 2 funding.</p>
-        <p>This kind of funding is only available to organisations whose ideas are mature enough to actually be in operation and being used by people.</p>
-        <p>For more information, please see the <a href="./eligibility">full detailed eligibility criteria</a> for this programme.</p>
-        <p>If you are at an earlier stage than this, we recommend looking at:</p>
-        <ul>
-          <li><a href="/funding/programmes">Our other funding programmes</a>
-          <li><a href="https://bethnalgreenventures.com/">Bethnal Green Ventures (external)</a>
-          <li><a href="https://socialtechtrust.org/">Social Tech Trust (external)</a>
-          <li><a href="https://www.comicrelief.com/">Comic Relief (external)</a>
-          <li><a href="https://www.unltd.org.uk/">UnLtd (external)</a>
-        </li>
+        eligibilitySummary:
+          introduction: "This strand of the Digital Fund is specifically designed to help organisations that:"
+          criteria:
+            - Your organisation must have been incorporated after the year 2000
```

Comment:
This doesn't make sense grammatically:

![image](https://user-images.githubusercontent.com/394376/46360363-259e6680-c663-11e8-88d7-519a234d5540.png)